.circleci/settings.py

```diff
@@ -0,0 +1,117 @@
+"""
+This script imports all of the required environment variables. If you are
+running it locally, update the .env file with the environment variables only
+available on your ci playform and run source.
+
+Usage: source .env && python3.6 initialize.py
+"""
+
+import os
+
+# PREFIX is the prefix of the name of the review apps created and deleted
+# by these scripts. PREFIX is used by `delete.py` to determine which apps
+# to delete after inactivity. So, it's important to use a prefix that won't
+# be used by Dash developers when creating apps manually, otherwise this
+# script may delete those apps!
+PREFIX = "review-app-"
+
+# LAST_UPDATE is the allowed amount of time before review apps are
+# deleted from the server.
+LAST_UPDATE = {"minutes": 1}
+
+# TRUNK_BRANCHNAME is the name of your repository's "main" or "master"
+# branch.
```

Comment:
In some places, it might not be called main or master, it might be called dev or trunk. So explain this variable more in terms of what it does in the script... what the implications are. Like "Set this to the trunk branch that you'd like to have update your target app. When `BRANCHNAME` IS `TRUNK_BRANCHNAME`, the script will deploy the changes to the target app. Otherwise, the script will create a new review app. This is usually "main" or "master""